src/main/java/cloud/fogbow/ras/core/cloudconnector/LocalCloudConnector.java

```diff
@@ -47,18 +47,20 @@
     private SecurityRulePlugin securityRulePlugin;
     private GenericRequestPlugin genericRequestPlugin;
 
+    private boolean shouldAuditRequests;
```

Comment:
I suggest changing the name of this field to auditRequestOn.

Reply:
Makes sense, renaming it.